Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -208,39 +209,25 @@ private class DisconnectSetting: WithAccountSetting {
 private class SyncNowSetting: WithAccountSetting {
     private let syncNowTitle = NSAttributedString(string: NSLocalizedString("Sync Now", comment: "Sync Firefox Account"), attributes: [NSForegroundColorAttributeName: UIColor.blackColor(), NSFontAttributeName: UIFont.systemFontOfSize(UIConstants.DefaultStandardFontSize, weight: UIFontWeightRegular)])
 
-    private let log = Logger.browserLogger
+    private let syncingTitle = NSAttributedString(string: NSLocalizedString("Syncing…", comment: "Syncing Firefox Account"), attributes: [NSForegroundColorAttributeName: UIColor.grayColor(), NSFontAttributeName: UIFont.systemFontOfSize(UIConstants.DefaultStandardFontSize, weight: UIFontWeightRegular)])
```

Comment:
Let's keep this as "Syncing…" for now. We haven't shipped and we're still string-frozen.